src/scripts/Data.js

```diff
@@ -0,0 +1,49 @@
+export const Works = [
```

Comment:
use `export default` to solve it

Reply:
![Screenshot from 2022-04-06 12-23-42](https://user-images.githubusercontent.com/75540413/161943146-a01d9003-bb4a-4b6b-be0c-0537bafbea74.png)
![Screenshot from 2022-04-06 12-24-37](https://user-images.githubusercontent.com/75540413/161943171-2da9432c-3935-481f-8ac5-56fc254c8b5d.png)

Reply:
# Please try the solution before advice